### PR TITLE
Education sector

### LINKED ISSUE #5: 🎨  Home Page Educational Section Component
#  Create the home Page "Educational Solution" section

Quick links: [Summary](#summary) | [Description](#description) | [Technical-Analysis](#Technical-Analysis) | [Acceptance-Criteria](#Acceptance-Criteria)

---

### Summary

As a user, I want a section that shows an Educational section that describes what Elewa does as shown in the image below

### Description

Design:

educational section

Educational section2

Link to full design:

```
https://xd.adobe.com/view/b98caf27-6549-49e5-8ed8-c5234788df61-ecdf/screen/cc35e19a-24f2-42ce-8803-ee2942839528
```
### Technical Analysis

This feature can be found in `index.html`

### Acceptance Criteria

- [ ] When rendered the section is pixel-perfect compared to the design above.
- [ ] The section is aligned using CSS grid
- [ ] The design is responsive for screen, tablet and phone

### PR DESCRIPTION
**Pull request by afrikhana**
# Description

Updated the page per the request on making it as friendly as possible.

Fixes # (issue) - *Example*: Fixes #5 

## Screenshot (optional)
education1
#updated laptop size
education-laptop1
education-laprop2
landscape-education
#tablet
landscape-ipad
smalleripad-education
potrate-ipad
smaller-education
#mobile
mobile-education
mobile-education1

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by JAPHETHNYARANGA — COMMENTED** [review]
Please push only the required files for the specific issue, and send a full screenshot  of both web and mobile view, showing   also the web browser.

File: src/css/style.css [hunk 1]
```diff
@@ -1,118 +1,482 @@
...
+    place-items: center;
+    text-align: justify;
+   }
+   .display .parent{
+    position: relative;
+    display: flex;
+    width: 450px;
+    flex-direction: row-reverse;
+    margin-left: -40px;
+    margin-top: -15%;
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: this could be improved

File: src/css/style.css [hunk 2]
```diff
@@ -1,118 +1,482 @@
...
+   
+   }
+   .display3 #oval3{
+    width: 450px;
+    height: 250px;
+    background-image: linear-gradient(to right, #f7d7b1, #f2ba7f);
+    border-bottom-left-radius: 200px;
+       border-top-left-radius: 200px;
+       border-bottom-right-radius: 200px;
+       border-top-right-radius: 200px;
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: use this border-radius: 200px ;  same values

File: src/pre.txt [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+span{
+    font-size: 1.7rem;
+    font-family: 'Roboto', sans-serif;
+}
+.wel-content button{
+    color: white;
+    background-color: black;
+    border: none;
+    margin-left: 150px;
+    margin-top: -30px;
```
Comment (JAPHETHNYARANGA) [inline src/pre.txt]: margin: -30px 0  0 150px;

File: src/pre.txt [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+    margin-top: 7px;
+
+}   
+.wel-content .material-symbols-outlined{
+width: 25px;
+background-color: white;
+color: black;
+border-radius: 50%;
+margin-left: 5px;
+margin-top: 2px;
```
Comment (JAPHETHNYARANGA) [inline src/pre.txt]: same

**Review by JAPHETHNYARANGA — COMMENTED** [review]
Good attempt, just those few changes , less than 5 minutes , test , then come tell me i merge.

File: src/pre.txt (hunk 4 above)
Comment (afrikhana) [reply src/pre.txt]: sr thats the pre text not html file...but i deleted it so that it doesnt cause any issue

**Comment by afrikhana** [comment]
Done with everything sr

**Review by JAPHETHNYARANGA — APPROVED** [review]
good attempt

**Pull request merged**